src/components/AppToolbar.tsx

```diff
@@ -134,6 +137,19 @@ class AppToolbarInternal extends React.Component<AppToolbarInternalProps> {
     }
   }
 
+  onFileSelected = (e: File[]) => {
+    const file = e[0];
+    this.props.onLocalPMTilesSelected(file);
+  }
+
+  onFileRejected = (r: FileRejection[]) => {
+    const errorMessageLine = r.map(e => {
+      return e.errors.map(f => f.message).join("\n")
+    }).join("\n");
+    console.error("Dropzone file rejected:", errorMessageLine);
```

Comment:
I don't see a reason to both print to the console and notify the user, so this can be removed I guess.